docs/Python-API.rst

```diff
@@ -12,6 +12,7 @@ Data Structure API
     Dataset
     Booster
     CVBooster
+    Sequence
 
 Training API
 ------------
```

Comment:
Please consider updating the following two methods:
https://github.com/microsoft/LightGBM/blob/c7134fa7ccbe1a1b1819ac2a66407504d3fab27a/python-package/lightgbm/basic.py#L1939
https://github.com/microsoft/LightGBM/blob/c7134fa7ccbe1a1b1819ac2a66407504d3fab27a/python-package/lightgbm/basic.py#L2048

Reply:
I've never used these two APIs. I need some time to figure out how to change these two methods.

Reply:
OK. I think it's totally fine to do this in a follow-up PR. I'll create an issue referring this comment to not lose our discussion after merging this PR.

Reply:
Created #4403 to track this enhancement.

Reply:
Thanks for the separate issue.